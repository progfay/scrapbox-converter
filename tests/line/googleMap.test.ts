/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('googleMap', () => {
  it('Simple googleMap', () => {
    const input = `Google Map
[N35.6972422,E139.7593459,Z17 明治大学]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
