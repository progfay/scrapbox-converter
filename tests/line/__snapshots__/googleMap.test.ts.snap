// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`googleMap Simple googleMap 1`] = `
"<article>
  <h1 class=\\"title\\">Google&nbsp;Map</h1>
  <section>
    <p><iframe class=\\"googlemap\\" src=\\"https://maps.google.co.jp/maps?output=embed&amp;ll=35.6972422,139.7593459&amp;z=17&amp;q=%E6%98%8E%E6%B2%BB%E5%A4%A7%E5%AD%A6\\" width=\\"620\\" height=\\"450\\" frameborder=\\"0\\" allowfullscreen=\\"true\\"></iframe></p>
  </section>
</article>"
`;
